backend/src/apiserver/storage/object_store.go

```diff
@@ -41,6 +41,7 @@ type ObjectStoreInterface interface {
 	GetFromYamlFile(o interface{}, filePath string) error
 	GetPipelineKey(pipelineId string) string
 	GetSignedUrl(bucketConfig *objectstore.Config, secret *v1.Secret, expirySeconds time.Duration, artifactURI string) (string, error)
+	GetSignedUrlWithoutContentDisposition(bucketConfig *objectstore.Config, secret *v1.Secret, expirySeconds time.Duration, artifactURI string) (string, error)
```

Comment:
Could we make this more configurable? For example, the following would allow further customizations without having to add a new method for each variation.
```suggestion
	GetSignedUrlWithQueryParams(bucketConfig *objectstore.Config, secret *v1.Secret, expirySeconds time.Duration, artifactURI string, queryParams url.Values) 
```

Reply:
This method isn't needed anymore since you can just call `GetSignedUrlWithQueryParams` with the correct content disposition header provided as a query parameter.